config/jobs/image-pushing/k8s-staging-sig-storage.yaml

```diff
@@ -407,8 +407,8 @@ postsubmits:
               - --scratch-bucket=gs://k8s-staging-sig-storage-gcb
               - --env-passthrough=PULL_BASE_REF
               - .
-  kubernetes-sigs/container-object-storage-interface-api:
-    - name: post-container-object-storage-interface-api-push-images
+  kubernetes-sigs/container-object-storage-interface-controller:
+    - name: post-container-object-storage-interface-controller-push-images
```

Comment:
Why removing kubernetes-sigs/container-object-storage-interface-api?

Reply:
I do not have images in the api repo to push. is my understanding correct?